HW 02/Task 2.3/PracticalLesson.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+namespace Task_2._3
+{
```

Comment:
public class PracticalLesson : Lesson. Do not add ICloneable here, it is inherited  from Lesson,